src/constants/messages.js

```diff
@@ -0,0 +1,45 @@
+import {
+  MAX_CAR_COUNT,
+  MIN_CAR_NAME_LEN,
+  MAX_CAR_NAME_LEN,
+  MIN_TRY_COUNT,
+  MAX_TRY_COUNT,
+} from './constants.js';
+
+const PRINT_MESSAGES = {
+  INPUT: {
+    CAR_NAME: '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)',
+    TRY_COUNT: '시도할 횟수는 몇 회인가요?',
+  },
+  OUTPUT: {
+    RESULT: '실행 결과',
+    WINNER: '최종 우승자',
+  },
+};
+
+const ERROR_MESSAGE = {
```

Comment:
상수 객체를 `Object.freeze()`로 동결하고 객체 내부에 여러 속성들이 있기 때문에 네이밍으로 복수인 것을 명시해도 좋을 것 같아요!
```suggestion
const ERROR_MESSAGES = Object.freeze({
```

Reply:
Object.freeze()로 동결하면 객체의 불변성을 유지할 수 있다는 측면에서 확실히 좋은 방법인 것 같습니다!! 덕분에 Object.freeze()에 대해서 찾아보게 되었는데 Object.freeze()는 얕은 동결을 수행하기 때문에 객체 내에 중첩된 객체까지 동결하려면 중첩 객체들도 각각 Object.freeze()로 동결해야 한다는 새로운 사실도 알게 되었습니다.🤓

위의 PRINT_MESSAGES의 경우 중첩된 객체이기 때문에 
```
Object.freeze(PRINT_MESSAGES.INPUT);
Object.freeze(PRINT_MESSAGES.OUTPUT);
Object.freeze(PRINT_MESSAGES);
```
이와 같이 freeze를 해야한다는 새로운 사실도 알게되는 리뷰였습니다. 좋은 의견 남겨주셔서 감사합니다!!